### PR TITLE
Bgrm

### PR DESCRIPTION
**Pull request by blueOkiris**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
    * Using `--devices=all` for access to webcam
    * Using `--filesystem=host` for users to be able to use custom backgrounds from their photos
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: https://github.com/blueOkiris/bgrm**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.blueokiris.bgrm.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                    "url": "https://www.python.org/ftp/python/3.10.4/Python-3.10.4.tar.xz",
+                    "sha256": "80bf925f571da436b35210886cf79f6eb5fa5d6c571316b73568343451f77a19"
+                }
+            ]
+        }, {
+            "name": "virtualenv",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
```
Comment (hfiguiere) [inline com.blueokiris.bgrm.json:27]: this won't work. builds are sanboxed.

Also you don't need a venv

File: com.blueokiris.bgrm.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+    "app-id": "com.blueokiris.bgrm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bgrm-fp.sh",
+    "finish-args": [
+        "--device=all",
```
Comment (hfiguiere) [inline com.blueokiris.bgrm.json:8]: you need --share=ipc for x11 (even in fallback)

**Review by hfiguiere — CHANGES_REQUESTED** [review]
There is no appstrean, desktop nor icon files.

Also the upstream source should not be included here.

**Comment by blueOkiris** [comment]
> There is no appstrean, desktop nor icon files.

It is a CLI application

**Comment by hfiguiere** [comment]
thats not even clear since you granted permissions to display.

Yes they are necessary.

**Comment by blueOkiris** [comment]
Eh. This is too hard to figure out. I'll just not have it on flathub

**Pull request closed without merging**